src/main_benchmark.cpp

```diff
@@ -28,6 +28,7 @@
 #include <stdio.h>
 #include <sysexits.h>
 
+#include "capi_frontend/capi_utils.hpp"
```

Comment:
I think we should be trying to get closer to making this benchmark app standalone, instead of enforcing users to compile it together with ovms (build image).
Therefore adding second OVMS header (previously only stringutils.hpp) is not good idea